### PR TITLE
API-001 Extends items endpoint

### PR DESCRIPTION
**Pull request by jakubgiminski**
Task: https://github.com/jakubgiminski/order-item/projects/1#card-5261472

### CODE REVIEW

**Comment by a-barzanti** [comment]
Comment test

File: src/endpoints.config.php [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 return [
     Endpoint::get('/items', ItemsController::class, 'getAll'),
+    Endpoint::get('/items?order_id={order_id}', ItemsController::class, 'getAllFromOrder'),
```
Comment (malexc) [inline src/endpoints.config.php]: End point should be /orders/{order_id}/items

Comment (jakubgiminski) [reply src/endpoints.config.php]: Why? :p

File: src/Item/Item.php [hunk 2]
```diff
@@ -9,13 +9,21 @@ class Item implements Entity
 {
     private $id;
 
-    public function __construct(string $id)
+    private $orderId;
```
Comment (dbojdo) [inline src/Item/Item.php]: Missing docblock

File: src/Item/ItemsRepository.php [hunk 3]
```diff
@@ -24,4 +26,15 @@ public function getAll(): ItemsCollection
 
         return $items;
```
Comment (dbojdo) [inline src/Item/ItemsRepository.php]: Unused code

**Pull request closed without merging**